src/MeetingsSDKAdapter.js

```diff
@@ -258,9 +264,9 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
       observer.complete();
     });
 
-    const muteEvent$ = fromEvent(sdkMeeting, EVENT_MEDIA_LOCAL_MUTE).pipe(
-      filter((event) => event.control === MUTE_AUDIO_CONTROL),
-      map(() => ({...muted, state: MeetingControlState.INACTIVE}))
+    const muteEvent$ = fromEvent(sdkMeeting, EVENT_MEDIA_LOCAL_UPDATE).pipe(
+      filter((event) => event.control === AUDIO_CONTROL),
+      map(({state}) => (state ? muted : unmuted))
     );
 
     return concat(getDisplayData$, muteEvent$);
```

Comment:
I can't comment on the exact line, but make sure that `getDisplayData$` send `unmuted` display as default. When we first load the control, it should not be activated